data/locale.yml

```diff
@@ -262,6 +262,10 @@ clipboard:
   copy_to_clipboard: Copy to Clipboard
   copied: Copied
 
+launch_button:
```

Comment:
Externalized strings! I like it. 🌏

Reply:
I actually don't know how they'll land in Crowdin - so i just followed the general paradigm 🚀